jobbergate-core/jobbergate_core/__init__.py

```diff
@@ -0,0 +1,13 @@
+"""
+Jobbergate-core contains key components that are shared among sub-projects.
+"""
+from .auth import AuthenticationError, JobbergateAuth, TokenError
```

Comment:
I think we should not use relative paths in projects. I would be much more comfortable with these paths appearing as:
```
from jobbergate_core.auth import ...
```

Using the full path does not much and it removes any possible ambiguity.

Reply:
solved in [7d94777](https://github.com/omnivector-solutions/jobbergate/pull/228/commits/7d9477776ec7fce9deea844be884ddcd06bef054)